users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -197,6 +205,8 @@ def get_returnn_config(
 
     if audio_perturbation:
```

Comment:
I meant more generally, that we wouldn't need to add the perturbation code for recognition. We either remove everything or nothing (since it does not impact the behavior during recognition, see below).